charts/hivemq-edge/tests/datahub/hivemq_edge_statefulset_test.yaml

```diff
@@ -105,6 +105,21 @@ tests:
           content:
             name: HIVEMQ_DATAHUB_ENABLED
             value: "true"
+      - contains:
+          path: spec.template.spec.containers[0].env
+          content:
+            name: HIVEMQ_DATAHUB_WAIT_BEFORE_DELETE
+            value: "5000"
+      - contains:
+          path: spec.template.spec.containers[0].env
+          content:
+            name: HIVEMQ_DATAHUB_WATCHER_INITIAL_DELAY
+            value: "2000"
```

Comment:
the initial delay of 2000 is too low. the default is 60000 

```
defaultValuesMap.put(DATA_HUB_PRESET_WATCHER_INITIAL_DELAY_MILLIS, "60000"); // 60 seconds
```

- Edge starts, finds the file and has HIVEMQ_DATAHUB_WATCHER_INITIAL_DELAY millis to process it and put it into persistence. Two seconds may be too narrow if the preset is large. The update task will kick in and this will result in a problem potentially.

There is another default:

```
defaultValuesMap.put(DATA_HUB_PRESET_WATCHER_INTERVAL_MILLIS, "60000"); // every 60 seconds
```

I think 2 seconds is too snappy. My mental use case is that users need a couple minutes to think, make changes, save. Checking for file changes every 2 seconds seems to be an excessive burn of cpu.

Reply:
Good points. 1 minute is still wuite long. How about 30 seconds?
About the watcher:
Changes are coming via Helm to this. People might roll this out a lot faster depending on what they are doing.
If the check is only doing a timestamp comparison on the file we should be able to reduce the interval to something around 5-10 seconds.